udata/harvest/api.py

```diff
@@ -257,6 +257,8 @@ def post(self, ident):
             data = request.json
         except Exception as e:
             data = request.data
+        if type(data) == bytes:
+            data = data.decode('utf-8')
```

Comment:
when do you have bytes and when do we not? probably when it's not json? if it is, we should cast it in the above catch block.

if it's not, we should catch that at a higher level of abstraction I think, like in `actions.schedule` 

there is already a `str()` cast there BTW https://github.com/opendatateam/udata/blob/master/udata/harvest/actions.py#L212, maybe it's the one causing trouble, because:

```
>>> a = b'a'
>>> a
b'a'
>>> str(a)
"b'a'"
```

Reply:
the str cast is there because the function signature accepts an hour, minute ect argument. Thoses arguments may be string or integers. This behavior can be found in the tests.
I will check if the bytes are retrieved only when it is not Json or not.